lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -612,8 +612,7 @@ LogicalResult iterativelyFuseProducerAndConsumerOfTiledOp(
 
 /// This is a workaround to deal with LinalgXOp
 static bool isTilableLinalgXOp(Operation *op) {
-  return isa<linalgx::BatchReduceMatmulVnniOp, linalgx::MultiBatchMatmulOp,
-             linalgx::Mm2DVnniOp, linalgx::Mm4DVnniOp>(op);
+  return linalgx::isAnyGenericPackedMatmulOp(op);
 }
 
```

Comment:
If there exist no `linalgX::matmul` any more. Please help to remove `isTilableLinalgXOp` definition and its caller.